apigw-websocket-api-lambda-authorizer/apigw-websocket-api-lambda-authorizer.json

```diff
@@ -0,0 +1,100 @@
+{
+    "title": "WebSocket API Cognito Authentication using Lambda Authorizer",
+    "description": "This pattern demonstrates how to integrate Cognito authentication with Amazon API Gateway WebSocket API.",
```

Comment:
Please change to "How to integrate Cognito authentication with Amazon API Gateway WebSocket API."